scripts/test/SANS/gui_logic/run_tab_presenter_test.py

```diff
@@ -267,6 +268,22 @@ def test_fails_silently_when_batch_file_does_not_exist(self):
         self._remove_files(user_file_path=user_file_path)
         self.os_patcher.start()
 
+    def test_batch_file_dir_not_added_to_config(self):
```

Comment:
Just for my own understanding why do we want the batch file dir not to be added here?

Reply:
The idea was that now we automatically add the path, if you accidentally click on the wrong directory/file, your manage directories would still have the wrong directory, even if loading the batch file failed because it wasn't valid. So the next time you ever had to manually manage directories it may be very busy with directories you didn't actually intend to add.

I'm not sure if this ever would be a problem in practice though, as I imagine the users don't often select batch files from wrong directories. I can remove if you think it's unecessary

Reply:
Ah ok could we change the test title to something like `test_batch_file_dir_not_added_to_config_if_batch_file_load_fails`